### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by mkrzyzanowski**
- introduced Action
- first action implementation: ReadResource
- utils related to testing (JUnit 5 and WireMock)

### CODE REVIEW

File: core/src/main/kotlin/com/cognifide/apmt/actions/Aem.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.cognifide.apmt.actions
+
+object Aem {
+    const val CSRF_ENDPOINT: String = "/libs/granite/csrf/token.json"
```
Comment (mjedraszczyk) [inline core/src/main/kotlin/com/cognifide/apmt/actions/Aem.kt]: You don't need to put constants into object. The may be defined directly in file.

File: core/src/main/kotlin/com/cognifide/apmt/junit/WireMockExtension.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.cognifide.apmt.junit
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.ParameterContext
+import org.junit.jupiter.api.extension.ParameterResolver
+
+class WireMockExtension : ParameterResolver {
+    override fun resolveParameter(parameterContext: ParameterContext?, extensionContext: ExtensionContext?): Any {
```
Comment (mjedraszczyk) [inline core/src/main/kotlin/com/cognifide/apmt/junit/WireMockExtension.kt]: Fix formmating here.

File: core/src/main/kotlin/com/cognifide/apmt/junit/Constants.kt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.cognifide.apmt.junit
+
+import org.junit.jupiter.api.extension.ExtensionContext
+
+object Constants {
+    const val WIREMOCK_SERVER: String = "wiremockServer"
```
Comment (mjedraszczyk) [inline core/src/main/kotlin/com/cognifide/apmt/junit/Constants.kt]: You may put vals directly in file.

File: core/src/main/kotlin/com/cognifide/apmt/junit/WireMockExtension.kt (hunk 2 above)
Comment (mkrzyzanowski) [reply core/src/main/kotlin/com/cognifide/apmt/junit/WireMockExtension.kt]: It's formatted according to the IntelliJ's Kotlin style guide :)

**Pull request merged**